packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/TryCatchFinallyStatement.cs

```diff
@@ -9,46 +9,35 @@ namespace Microsoft.Generator.CSharp.Statements
 {
     public sealed class TryCatchFinallyStatement : MethodBodyStatement
     {
-        public MethodBodyStatement Try { get; }
-        public IReadOnlyList<CatchExpression> Catches { get; }
-        public MethodBodyStatement? Finally { get; }
+        public TryStatement Try { get; }
+        public IReadOnlyList<CatchStatement> Catches { get; }
+        public FinallyStatement? Finally { get; }
 
-        public TryCatchFinallyStatement(MethodBodyStatement tryStatement, IReadOnlyList<CatchExpression> catches, MethodBodyStatement? finallyStatement)
+        public TryCatchFinallyStatement(TryStatement @try, IReadOnlyList<CatchStatement> catches, FinallyStatement? @finally)
         {
-            Try = tryStatement;
+            Try = @try;
             Catches = catches;
-            Finally = finallyStatement;
+            Finally = @finally;
         }
 
-        public TryCatchFinallyStatement(MethodBodyStatement Try) : this(Try, Array.Empty<CatchExpression>(), null)
+        public TryCatchFinallyStatement(TryStatement @try) : this(@try, Array.Empty<CatchStatement>(), null)
         {
         }
 
-        public TryCatchFinallyStatement(MethodBodyStatement Try, CatchExpression Catch, MethodBodyStatement? Finally = null) : this(Try, [Catch], Finally)
+        public TryCatchFinallyStatement(TryStatement @try, CatchStatement catchStatement, FinallyStatement? @finally = null) : this(@try, new[] { catchStatement }, @finally)
```

Comment:
name `tryStatement` and `finallyStatement` to match `catchStatement`?

Reply:
I meant to rename all to the shorter form.